http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -187,7 +187,6 @@ object KubernetesServiceDbQueries {
 
   def markPreDeleting(nodepoolId: NodepoolLeoId, appId: AppId)(implicit ec: ExecutionContext): DBIO[Unit] =
     for {
-      _ <- nodepoolQuery.updateStatus(nodepoolId, NodepoolStatus.Predeleting)
```

Comment:
Can we remove the `NodepoolStatus.Predeleting` enum altogether?

Would also be good to update the appropriate tests in `KubernetesServiceInterpSpec` to make sure the nodepool status is still `RUNNING` after an app is deleted.

Reply:
I can do this also in this PR

Reply:
@rtitle [info] - should delete an app and update status appropriately *** FAILED ***
[info]   PRECREATING did not equal RUNNING (KubernetesServiceInterpSpec.scala:268)

Hm, getting this failure on the unit tests (changed the line to Running). This seems really strange to me since this implies that the status is being changed solely with the deleteApp call. I don't think we've had issues with this test recently, maybe there's another bug here.

Reply:
I think `precreating` is expected here because the test does not include a back leo/pubsub subscriber. The test does:

1. `kubeServiceInterp.createApp`
(app and nodepool are `precreating` at this point)
2. manually update app to `running` (simulating back leo)
3. `kubeServiceInterp.deleteApp` 
(app is `predeleting` now and nodepool status should not change)

Maybe you can modify step (2) to update _both_ app and nodepool status to Running for the sake of simulating back Leo.